src/get-definitely-typed.ts

```diff
@@ -225,7 +225,7 @@ class DiskFS implements FS {
     }
 
     readdir(dirPath?: string): ReadonlyArray<string> {
-        return assertSorted(readdirSync(this.getPath(dirPath))).filter(name => name !== ".DS_STORE");
+        return assertSorted(readdirSync(this.getPath(dirPath)).sort()).filter(name => name !== ".DS_STORE");
```

Comment:
Can you get rid of the `assertSorted` call? Now, it's just testing that `sort` works.

Reply:
Done.